docs/reference/3-admin-api/technical-articles/admin-api-1x-docker-microsoft-sql-server-support.md

```diff
@@ -0,0 +1,176 @@
+# Admin API 1.x Docker & Microsoft SQL Server Support
+
+**Pre-requisites:**
+
+* SQL Server exposed or locally
+
+* SQL User different from sa
+
+* Clone [Admin Api 1.x](https://github.com/Ed-Fi-Alliance-OSS/AdminAPI-1.x)
+    repository
+
+* Generate the certificate for nginx by running the
+    [Docker/Settings/ssl/generate-certificate.sh](https://github.com/Ed-Fi-Alliance-OSS/AdminAPI-1.x/tree/main/Docker/Settings/ssl)
+    script
+
+**File configuration.**
+
+1. Set the version of AdminApi to use in
+    [Docker/Dockerfile](https://github.com/Ed-Fi-Alliance-OSS/AdminAPI-1.x/blob/main/Docker/Dockerfile)
+    Update the parameters:
+    **DB**: mssql
+    **VERSION**: 1.4.1
+
+    ```
```

Comment:
Can we remove the sample Dockerfile below, since we have a link to the contents above? That way we don't need to keep this content below up to date. In fact, the example below installs postgresql support on line29 unnecessarily.